app/src/main/java/com/github/globant/githubsubscribers/commons/models/Subscriber.java

```diff
@@ -0,0 +1,164 @@
+package com.github.globant.githubsubscribers.commons.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.lang.reflect.Field;
+
+/**
+ * Subscriber class that represents each Github subscriber
+ *
+ * @author edwin.cobos
+ * @author juan.herrera
+ * @since 19/08/2016
+ */
+public class Subscriber {
+
+    private final String login;
+    private final String id;
+    @SerializedName("avatar_url")
+    private final String avataUrl;
+    @SerializedName("gravatar_id")
+    private final String gravatarId;
+    private final String url;
+    @SerializedName("html_url")
+    private final String htmlUrl;
+    @SerializedName("followers_url")
+    private final String followersUrl;
+    @SerializedName("following_url")
+    private final String followingUrl;
+    @SerializedName("gists_url")
+    private final String gistsUrl;
```

Comment:
Do we need to parse every single attribute from the API for the given requirements? I think gistsUrl as well as starredUrl, eventsUrl, etc. are not needed.

What is the purpose of having them?